blocks/about/about.css

```diff
@@ -0,0 +1,18 @@
+@import url(./__text/about__text.css);
+@import url(./__italic-text/about__italic-text.css);
+
+.about {
+  margin: 68px 30px 0;
```

Comment:
It's better to use padding instead of margin for blocks

Reply:
Done